notebooks/JWST/NIRCam/PSF_phot_01/GriddedPSFModel_simulation_and_photometry.ipynb

```diff
@@ -0,0 +1,552 @@
+    "## Introduction\n",
+    "\n",
+    "The simulated image will be made using a spatially-dependent JWST PSF generated by `webbpsf`.  `photutils` will be then used to perform PSF-fitting photometry on the simulated JWST NIRCam image.\n",
+    "\n",
+    "The intended audience are astronomers already familiar with PSF photometry, the FITS file format, and basic Python usage. In-depth scientific and technical backgrounds will not be discussed nor taught in this notebook.\n",
+    "\n",
+    "**Background on JWST PSF simulation:** https://jwst.stsci.edu/science-planning/proposal-planning-toolbox/psf-simulation-tool-webbpsf\n",
+    "\n",
+    "**Background on PSF photometry:** https://photutils.readthedocs.io/en/stable/psf.html\n",
+    "\n",
+    "**Some notes about the file:** It is pre-generated grid of JWST PSFs for NIRCam A1 detector in F090W filter. The file is hosted on STScI Box file sharing system.\n",
+    "\n",
+    "**Defining some terms:**\n",
+    "\n",
+    "* **F090W:** A NIRCam filter\n",
+    "* **FITS:** Flexible Image Transport System (https://fits.gsfc.nasa.gov/fits_primer.html)\n",
+    "* **HDU:** Header/Data Unit in a FITS file\n",
+    "* **JWST:** James Webb Space Telescope\n",
+    "* **NIRCam:** Near-Infrared Camera on JWST (https://jwst.stsci.edu/instrumentation/nircam)\n",
+    "* **NRCA1:** NIRCam A1 detector\n",
+    "* **PSF:** Point-spread function"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading data\n",
+    "\n",
+    "We start by loading into FITS HDU a pre-generated grid of JWST PSFs for NIRCam A1 detector in F090W filter."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "url = 'https://stsci.box.com/shared/static/6h8wsr2ts0t24s79cxmnyk8bldt0vv3i.fits'\n",
+    "hdu = fits.open(url)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## File information"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hdu.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Astropy has saved a cached copy of this file under your local cache directory for Astropy; default is `~/.astropy/cache/download`.\n",
+    "\n",
+    "This file only has a single extension that holds a data cube. The cube consists of 16 PSF images, each with dimension of 404 by 404 pixels. Each PSF is oversampled by a factor of 4, so the actual detector pixel dimension covered by the PSF is 101 by 101.\n",
+    "\n",
+    "Now, let's look at the first PSF in the cube."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = hdu[0].data[0]\n",
+    "norm = simple_norm(data, 'sqrt', percent=99.)\n",
+    "plt.figure(figsize=(9, 9))\n",
+    "plt.imshow(data, norm=norm)"
```

Comment:
Maybe this (and the above ``norm=...`` line) should be replaced with the `imshow_norm` function in `astropy.visualization`?  E.g.:
```suggestion
    "imshow_norm(data, interval=PercentileInterval(99.), stretch=SqrtStretch())"
```
and similar with the other uses of norm + imshow?

(Note that along with the above suggestion, would need to delete the `norm` and add ``from astropy.visualization import imshow_norm, PercentileInterval, SqrtStretch`` at the top.

Reply:
Done. But you might wanna double check the rendered artifact, as I am editing the notebook directly on GitHub. 😨